lib/Fhp/BaseAction.php

```diff
@@ -39,6 +41,9 @@ abstract class BaseAction implements \Serializable
     /** @var int[] Stores segment numbers that were assigned to the segments returned from {@link createRequest()}. */
     private $requestSegmentNumbers;
```

Comment:
Vielleicht brauchen wir dann $requestSegmentNumbers nicht mehr separat, denn die Nummern wären ja schon in $message enthalten.